libs/api/domains/auth/src/lib/resolvers/delegation.resolver.ts

```diff
@@ -41,20 +41,20 @@ export class DelegationResolver {
   constructor(private authService: AuthService) {}
 
   @Query(() => [Delegation], { name: 'authActorDelegations' })
-  getActorDelegations(@CurrentUser() user: User): Promise<Delegation[]> {
+  getActorDelegations(@CurrentUser() user: User): Promise<DelegationDTO[]> {
     return this.authService.getActorDelegations(user)
   }
 
   @Query(() => [Delegation], { name: 'authDelegations' })
-  getDelegations(@CurrentUser() user: User): Promise<Delegation[]> {
+  getDelegations(@CurrentUser() user: User): Promise<DelegationDTO[]> {
     return this.authService.getDelegations(user)
   }
 
   @Query(() => Delegation, { name: 'authDelegation', nullable: true })
   async getDelegation(
     @CurrentUser() user: User,
     @Args('input', { type: () => DelegationInput }) input: DelegationInput,
-  ): Promise<Delegation | null> {
+  ): Promise<DelegationDTO | null> {
     const delegation = await this.authService
```

Comment:
The initial idea with these objects were that the DTO's were the request objects and the entities would be the response objects. Why are we returning DTO's now?